draft-mt-ufmrg-teep-sample.md

```diff
@@ -87,6 +87,17 @@ modeling problem for slightly different reasons than those proposed in
    learning or training exercise, whereas {{?I-D.farrell-ufmrg-sample}}
    limits itself to just the `SEARCH` command of {{?RFC9051}}.
 
+4. The TEEP protocol follows a typical protocol design in the IETF where
+   various already standardized technologies are re-used. The architecture
+   of the TEEP protocol is also representative for IETF protocol development
+   since more than two parties are involved in the communication even in a
```

Comment:
Isn't this always the case in a protocol, as opposed to an algorithm, say, or a format specification?  Is your point here that there are multiple participants in the protocol, whose interactions can therefore be modeled under different scenarios?  (Whereas IMAP `SEARCH` is just client-server?)

Reply:
I enhanced the text to provide clarifications.